### PR TITLE
cmd/action: support sqlite

### PR DESCRIPTION
**Pull request by yonidavidson**

### CODE REVIEW

File: cmd/action/mux.go [hunk 1]
```diff
@@ -76,7 +79,28 @@ func schemaNameFromDSN(url string) (string, error) {
...
+	drv, err := sqlite.Open(db)
+	if err != nil {
+		return "", err
+	}
+	r, err := drv.InspectRealm(context.Background(), nil)
+	if err != nil {
+		return "", err
+	}
+	if len(r.Schemas) != 1 {
+		return "", fmt.Errorf("number of schemas larger then 1, n=%d", len(r.Schemas))
```
Comment (a8m) [inline cmd/action/mux.go:103]:
```suggestion
		return "", fmt.Errorf("number of schemas larger than 1, n=%d", len(r.Schemas))
```

or just > 1

File: cmd/action/provider.go [hunk 2]
```diff
@@ -45,3 +47,19 @@ func postgresProvider(dsn string) (*Driver, error) {
...
+	}
+	drv, err := sqlite.Open(db)
+	if err != nil {
+		return nil, err
+	}
+	return &Driver{
+		Driver:      drv,
+		Marshaler:   sqlite.MarshalHCL,
+		Unmarshaler: sqlite.UnmarshalHCL,
+		Types:       sqlite.TypeRegistry.Specs(),
```
Comment (a8m) [inline cmd/action/provider.go:63]: Do we still need this after adding the UnmarshalHCL ones?

File: internal/integration/sqlite_test.go [hunk 3]
```diff
@@ -422,6 +426,34 @@ create table atlas_defaults
 	})
 }
 
+func TestSQLite_CLI(t *testing.T) {
+	h := `
+			schema "main" {
+				charset   = "%s"
+				collation = "%s"
```
Comment (a8m) [inline internal/integration/sqlite_test.go:433]: pretty sure this is mysql specific

File: internal/integration/sqlite_test.go [hunk 4]
```diff
@@ -792,3 +824,24 @@ func (t *liteTest) dropTables(names ...string) {
...
 	})
 }
+
+// defaultConfig returns the default charset and
+// collation configuration for testing.
+func (t *liteTest) defaultAttrs() []schema.Attr {
+	var (
+		charset   = "latin1"
+		collation = "latin1_swedish_ci"
+	)
```
Comment (a8m) [inline internal/integration/sqlite_test.go:834]: ditto

File: cmd/action/provider.go (hunk 2 above)
Comment (yonidavidson) [reply cmd/action/provider.go:63]: It's in all the other providers

**Review by a8m — APPROVED** [review]
looks good, but please change os.Mkdir to `strings.Replace("/", "_").

**Pull request merged**